archive/g/go/string-reverse.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import "fmt"
+import "os"
+
+//Function to reverse a string 
+func reverse_string(str string) string {
+	chars := []rune(str)
+	for i, j := 0, len(chars)-1; i < j; i, j = i+1, j-1 {
+		chars[i], chars[j] = chars[j], chars[i]
+	}
+	return string(chars)
+}
+
+func main() {
+	// Check the command line args length
+	var argslen = len(os.Args)
+	//fmt.Println(argslen)
+	
+	// If the length is 2, one command line arg exist
+	if argslen == 2 {
+		input := os.Args[1]
+		fmt.Println("Input: " , input)
+		fmt.Printf("Reverse: %v\n", reverse_string(input))
```

Comment:
For consistency, can we update this to output only the reversed string? (and not the input or "Reverse:")

Reply:
Good point! I must not have noticed this the first time.